devTools/cypress/integration/kitchenSink.js

```diff
@@ -159,11 +159,12 @@ describe("Kitchen sink", function () {
             title: "Cumulative confirmed COVID-19 cases",
             src: DEFAULT_THUMBNAIL_URL,
         })
-        testProminentLink({
-            testTitle:
-                "[NOT RECOMMENDED] Grapher link redirected to grapher (in wordpress)",
-            title: "Adjusted net savings per capita",
-            src: "http://localhost:3030/grapher/exports/adjusted-net-savings-per-person.svg",
-        })
+        // N/A
+        // testProminentLink({
+        //     testTitle:
+        //         "[NOT RECOMMENDED] Grapher link redirected to grapher (in wordpress)",
+        //     title: "Adjusted net savings per capita",
+        //     src: "http://localhost:3030/grapher/exports/adjusted-net-savings-per-person.svg",
+        // })
```

Comment:
does this test need updating?

Reply:
Seems so, yeah. I no longer have `adjusted-net-savings-per-person` in my exports after running a local build. We should probably try to not have these tests rely on the state of someone's local development DB.

Reply:
Yes could you please revert this so I'll take a look next time I run this and update the test?